heartbeat/cmd/root.go

```diff
@@ -34,7 +37,11 @@ var Name = "heartbeat"
 var RootCmd *cmd.BeatsRootCmd
 
 func init() {
-	RootCmd = cmd.GenRootCmdWithSettings(beater.New, instance.Settings{Name: Name})
+	settings := instance.Settings{
```

Comment:
Don't remember anymore the full affect of this. Does it only remove the `host.*` name fields or any other fields too?

Reply:
This should just remove the default `host.name` field. In comparison to this line, which I think the default for a beat, the difference is just the lack of `WithHost`.

https://github.com/elastic/beats/blob/ec7a9a2fb144ce2d80ee9c23d923a43dd6608551/libbeat/publisher/processing/default.go#L79

Reply:
That's my understanding as well